docs/tutorial/fdfitting.rst

```diff
@@ -62,39 +59,24 @@ We can simulate the model by passing a dictionary with parameters values::
 
     dna = lk.ewlc_odijk_distance("DNA")
     force = np.arange(0.1, 14, 0.1)
-    dna(force, {"DNA/Lp": 50.0, "DNA/Lc": 16.0, "DNA/St": 1500.0, "kT": 4.11})
+    distance = dna(force, {"DNA/Lp": 50.0, "DNA/Lc": 16.0, "DNA/St": 1500.0, "kT": 4.11})
+    plt.plot(distance,force)
+    plt.ylabel("Force")
+    plt.xlabel("Distance")
+
 
-Note how the model name is prefixed to the model specific parameters, forming a key-value pair like so: `"name/parameter": value`.
 
 Model composition and inversion
 -------------------------------
 
-In practice, we would typically want to fit force as a function of distance however. For this we have the inverted
-Odijk model::
+Above, we investigated a model with distance as a function of force. In practice, the force is typically fitted as a function of distance. This can be done using :func:`lk.ewlc_odijk_force() <lumicks.pylake.ewlc_odijk_force()>`::
```

Comment:
```suggestion
Above, we investigate a model with distance as a function of force. In practice, the force is typically fitted as a function of distance. This can be done using :func:`lk.ewlc_odijk_force() <lumicks.pylake.ewlc_odijk_force()>`::
```